charts/matrix-server/values.yaml

```diff
@@ -28,3 +28,5 @@ emailPattern: |-
 
 autojoinRooms:
 - "#arrivals:web3.foundation"
+
+antiscam: true
```

Comment:
Same as above, the default should be false and it should be set to true in the community matrix server chart values, here https://github.com/w3f/community-matrix-server/blob/master/helmfile.d/10-matrix.yaml#L13